src/NServiceBus.Transport.SQS/Extensions/MessageExtensions.cs

```diff
@@ -43,7 +43,7 @@ static class MessageExtensions
 
             using (var memoryStream = new MemoryStream())
             {
-                await s3GetResponse.ResponseStream.CopyToAsync(memoryStream).ConfigureAwait(false);
+                await s3GetResponse.ResponseStream.CopyToAsync(memoryStream, 81920, cancellationToken).ConfigureAwait(false);
```

Comment:
Does anyone see any problem with this? There is no overload of the method which accepts just a stream and a token. According to the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.copyto?view=netcore-2.1#System_IO_Stream_CopyTo_System_IO_Stream_), 81920 is the default buffer size.